src/components/Footer.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { FaInstagram, FaFacebookSquare, FaLinkedin } from "react-icons/fa";
+import { IoMail } from "react-icons/io5";
+import Image from "next/image";
+import icon from "../../public/images/navlogo.webp";
+
+const Footer = () => {
+  return (
+    <div className="bg-gradient-to-r from-isa-blue-200 to-isa-blue-100 w-full p-4">
+      <div className="flex items-center font-bold text-3xl text-white space-x-3">
+        <Image src={icon} alt="ISA icon" className="h-[45px] w-[45px]" />
+        <div className="whitespace-nowrap">UCR ISA</div>
+        <div className="w-full max-h-full flex space-x-4 place-content-end ">
+          <a href="https://www.instagram.com">
```

Comment:
You should map the social media links instead.
You can create an array of objects in the data file where each object contains the icon tag and a link.

Reply:
Here's an example of the mapping: https://acm-ucr.github.io/docusaurus/docs/react/mapping-filtering